sublime-package.json

```diff
@@ -315,10 +315,15 @@
               "default": false,
               "markdownDescription": "Show symbol action links in hover popup if available."
             },
-            "only_show_lsp_completions": {
+            "inhibit_explicit_completions": {
               "type": "boolean",
               "default": false,
-              "markdownDescription": "Disable Sublime Text's explicit and word completion."
+              "markdownDescription": "Disable Sublime Text's snippet completions."
+            },
+            "inhibit_word_completions": {
+              "type": "boolean",
+              "default": false,
```

Comment:
The `default` here doesn't match reality but I'm thinking now that the default should probably be the opposite of what is the actual default since when the user is overriding an option, he normally wants to override it to be an opposite of default.